solr/core/src/java/org/apache/solr/search/facet/FacetParser.java

```diff
@@ -16,18 +16,22 @@
  */
 package org.apache.solr.search.facet;
 
+import java.util.List;
+import java.util.ArrayList;
+import java.util.Map;
+import java.util.Optional;
+
 import org.apache.solr.common.SolrException;
 import org.apache.solr.common.params.SolrParams;
 import org.apache.solr.common.util.NamedList;
 import org.apache.solr.common.util.StrUtils;
+import org.apache.solr.search.FunctionQParser;
```

Comment:
Strange; I wonder why you (your IDE) re-ordered FunctionQParser to a different spot that is no longer alphabetic?  I don't care much about this so you can leave it but it's curious.

Reply:
Haven't the faintest clue whether it was the IDE or me reconciling the imports....